account_loan/security/ir.model.access.csv

```diff
@@ -7,3 +7,4 @@ access_account_loan_generate_wizard,access_account_loan_generate_wizard,model_ac
 access_account_loan_pay_amount,access_account_loan_pay_amount,model_account_loan_pay_amount,account.group_account_user,1,0,0,0
```

Comment:
wouldn't be better to set here 1,1,1,0 and not adding the new line?

Reply:
what about if manager needs to delete for any reason?

Reply:
it's a wizard, there is no need to delete :wink:

Reply:
ah ok